_includes/tutorials/fk-joins/kstreams/code/tutorial-steps/dev/albums.json

```diff
@@ -0,0 +1,4 @@
+5:{"id": 5, "title": "Physical Graffiti", "artist": "Led Zeppelin", "genre": "Rock"}
+6:{"id": 6, "title": "Highway to Hell",   "artist": "AC/DC", "genre": "Rock"}
```

Comment:
This is the correct format, it allows for passing in keys to `kafka-avro-console-producer`